sdk/storage/azblob/blob/client.go

```diff
@@ -525,7 +521,7 @@ func (b *Client) downloadFile(ctx context.Context, writer io.Writer, o downloadO
 		OperationName: "downloadBlobToWriterAt",
 		TransferSize:  count,
 		ChunkSize:     o.BlockSize,
-		NumChunks:     numChunks,
+		NumChunks:     uint16(numChunks),
 		Concurrency:   o.Concurrency,
 		Operation: func(ctx context.Context, chunkStart int64, count int64) error {
 			buff, err := acquireBuffer()
```

Comment:
We can directly get the buffer from channel here instead of making a func call.
```suggestion
			buff <- buffers.Acquire()
```